docs/lob_guide/src/asciidoc/_chapters/work_with_lob.adoc

```diff
@@ -0,0 +1,738 @@
+////
+/**
+* @@@ START COPYRIGHT @@@
+*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+* @@@ END COPYRIGHT @@@
+*/
+////
+
+[#work with lob]
+= Work with LOB
+
+[#create a sql table with lob columns]
+== Create a SQL Table with LOB Columns
+
+When creating a SQL table with LOB columns, following relevant tables and files are created as well:
+
+* One LOB MD table.
+* Two dependent descriptor tables.
+* HDFS data file (locates at /user/trafodion/lobs) for each column.
+
+[#syntax]
+== Syntax
+
+```
+CREATE TABLE table-name (lob-column-spec[, lob-column-spec]…)

```

Comment:
This syntax suggests that when you have lob columns in a table, you can't have other columns as well. Perhaps something like "CREATE TABLE table-name ( column_spec [, column_spec] ... [, primary_key_spec] )", then "column_spec is: lob-column-spec | other column spec"

Reply:
Thanks Dave! :)

I'm planning to change it to:

CREATE TABLE table-name (column-spec[, column-spec]…)

column-spec is:
lob-column-spec          | 
other-column-spec

lob-column-spec is:
column-name {lob-data-type} [column-constraint]

other-column-spec is:
column-name {data-type} [column-constraint]

lob-type is:
BLOB | CLOB [({numeric literal} [unit])] [STORAGE 'storage literal']

...

Any comments?

Reply:
I think this is fine.

Reply:
Thanks Dave! :)
I've corrected.